app-libs/stf/src/stf_sgx_primitives.rs

```diff
@@ -20,8 +20,11 @@ use itp_types::H256;
 use std::prelude::v1::*;
 
 pub mod types {
+	#[cfg(feature = "sgx")]
 	pub use sgx_runtime::{Balance, Index};
+	#[cfg(feature = "sgx")]
 	pub type AccountData = balances::AccountData<Balance>;
+	#[cfg(feature = "sgx")]
 	pub type AccountInfo = system::AccountInfo<Index, AccountData>;
```

Comment:
only these parts are restricted to `sgx`, the rest can be made available in `std` as well

Reply:
because they're type definitions from `sgx_runtime`, which is `sgx` only

Reply:
Ah. And we only use it in case we talk to the sgx-runtime. Got it. Thanks

Reply:
Well that too, yes. But even if I wanted, I couldn't make it `std`, because the types `Balance` and `Index` are from the `sgx_runtime` and used to define `AccountData` and `AccountInfo`. So it straight out won't compile in `std`.

Reply:
Ah, we should improve that: https://github.com/integritee-network/sgx-runtime/issues/37